requirements/base.in

```diff
@@ -1,4 +1,4 @@
 # Core requirements for using this application
 
 Django>=1.8,<2.0          # Web application framework
```

Comment:
We don't support anything below 1.11 anymore; could you update that here and in the cookiecutter?

Reply:
Sure thing